consensus/src/block_storage/block_tree.rs

```diff
@@ -448,12 +447,6 @@ impl BlockTree {
             committed_round = committed_round,
             block_id = block_to_commit.id(),
         );
-        event!("committed",
```

Comment:
it looks good to me but I am not very clear about this part change

Reply:
This is the special out-of-bound health check thing that CT used that we've been wanting to get rid of for a long time